main.py

```
@@ -129,19 +182,80 @@ def _save_disabled(self) -> None:
+
+        def _on_done(path: str) -> None:
+            progress_dlg.close()
+            reply = QMessageBox.question(
+                self, "Ready to Install",
+                f"{self._latest_version} downloaded.\n\n"
+                "JobDocs will close and the installer will launch.\nReady to install?",
+                QMessageBox.StandardButton.Yes | QMessageBox.StandardButton.No,
+            )
+            if reply == QMessageBox.StandardButton.Yes:
+                try:
+                    subprocess.Popen([path])
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)